### ISSUE TITLE
walkdir tries to parse nonexisting arg

### ISSUE THREAD

**Issue by matthiaskrgr** [Nilstrieb/cargo-minimize#3]
in the shell: `cargo minimize`

```
 WARN cargo_minimize::processor Error during walkdir: IO error for operation on src: No such file or directory (os error 2)
ERROR cargo_minimize An error occured:
Did not find any files for path src
```
Need to check if we actually pass any args or try `cwd` perhaps?

**Comment by Nilstrieb** [comment]
By default, it searches for rust files in `./src`. I guess you don't have that so it errors out (the error could probably be better). You can pass the path of the rust files that svojld be minimized as an argument

**Comment by Nilstrieb** [comment]
Using `./src` is mostly arbitrary, it may make more sense to just use the current working directory now that I think about it

**Comment by matthiaskrgr** [comment]
Ah, so I was going to just use some arbitrary folder, cp my .rs file in there and then use `--rustc` (without using anything cargo)

**Comment by Nilstrieb** [comment]
`--rustc` should just error out when the path isn't passed explicitly, that would make this obvious.

**Comment by Nilstrieb** [comment]
Fixed in [a866667](https://github.com/Nilstrieb/cargo-minimize/commit/a866667545437aa988525d2e4681d3f137fe11fc)
